hosting/src/firebase.ts

```diff
@@ -22,4 +23,9 @@ const emulatorDetails = {
 if (process.env.REACT_APP_ENV !== "production") {
 	connectAuthEmulator(getAuth(), emulatorDetails.authURL!);
 	connectFirestoreEmulator(getFirestore(), emulatorDetails.firestoreHost!, +emulatorDetails.firestorePort!);
+} else {
+	initializeAppCheck(app, {
+		provider: new ReCaptchaV3Provider("6LenhPoZAAAAAIE_6OQ4Aj6FwFd531NcLxh_a386"),
```

Comment:
What will happen locally?

Reply:
We're using the Emulator so App Check has no use - it will bypass. If we instantiate it locally it will cause React to crash anyway due to local host not being whitelisted in ReCaptcha.

Reply:
Ah yeah ignore me I see the prod check